source/system/structs/HashMap.ooc

```diff
@@ -193,6 +211,19 @@ HashMap: class <K, V> {
 		}
 		oldBuckets free()
 	}
+	writeToFile: func (filename: String) {
```

Comment:
`filename` is a `String` while `filename` in `readFromFile` is a `Text`? I would expect both methods to take the same type.